### PR TITLE
Create Tables and Queues if they don't exist

### PR DESCRIPTION
**Pull request by asampatoor**
@josephverbeck 

Have checked on my local. Tables and Queues are getting created.

### CODE REVIEW

File: config/initializers/aws.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+Aws.config.update({
+                      endpoint: ENV['aws_dynamo_endpoint'],
+                      access_key_id: Rails.application.secrets.aws_access_key_id,
+                      secret_access_key: Rails.application.secrets.aws_secret_access_key,
+                      region: ENV['aws_region']
+                  })
```
Comment (raseel-mohamed) [inline config/initializers/aws.rb]: add EOL space

File: config/initializers/shoryuken.rb [hunk 2]
```diff
@@ -26,12 +26,41 @@ def self.initialize_logger(log_target = STDOUT)
       @logger
     end
   end
+
```
Comment (raseel-mohamed) [inline config/initializers/shoryuken.rb]: :nit: extra space

File: config/initializers/shoryuken.rb [hunk 3]
```diff
@@ -26,12 +26,41 @@ def self.initialize_logger(log_target = STDOUT)
...
+    Shoryuken.logger.info "====== Creating queue [#{@queue_name}] ======"
+    @client.create_queue({:queue_name => @queue_name})
+    Shoryuken.logger.info "====== Created queue [#{@queue_name}] ======"
+  end
 end
 
 logger = Shoryuken::Logging.initialize_logger("#{Rails.root}/log/shoryuken.log")
 logger.level = Logger::DEBUG
 Rails.logger = logger
+
```
Comment (raseel-mohamed) [inline config/initializers/shoryuken.rb]: :nit: extra space. 

use rubocop gem to better lint your code. you can` configure it, but most of the style guides mentioned in https://github.com/bbatsov/ruby-style-guide is implemented here.

File: config/initializers/shoryuken.rb [hunk 4]
```diff
@@ -62,5 +61,4 @@ def self.initialize_logger(log_target = STDOUT)
 
 logger = Shoryuken::Logging.initialize_logger("#{Rails.root}/log/shoryuken.log")
 logger.level = Logger::DEBUG
-Rails.logger = logger
-
+Rails.logger = logger
```
Comment (raseel-mohamed) [inline config/initializers/shoryuken.rb]: I think there was a space the first time around Now there is EOL space missing now.  :)

Comment (asampatoor) [inline config/initializers/shoryuken.rb]: @raseel-mohamed

**Comment by raseel-mohamed** [comment]
You have a 👍 from me. Just remember to squash before merging.

**Pull request closed without merging**